lib/util.js

```diff
@@ -38,15 +38,15 @@ const inspectDefaultOptions = Object.seal({
   breakLength: 60
 });
 
-const CIRCULAR_ERROR_MESSAGE = 'Converting circular structure to JSON';
+const CIRCULAR_ERROR_MESSAGE = /Circular/i;
 
 var Debug;
 
 function tryStringify(arg) {
   try {
     return JSON.stringify(arg);
   } catch (err) {
-    if (err.name === 'TypeError' && err.message === CIRCULAR_ERROR_MESSAGE)
+    if (err.name === 'TypeError' && err.message.match(CIRCULAR_ERROR_MESSAGE))
```

Comment:
I don't really like this particular change because it makes the test less specific, even if we are unlikely to get burned by it. I think the `common.engineSpecificMessage` API is a good solution in the long term. In the shorter term, you could generate this particular error dynamically like this:

```js
try { const a = {}; a.a = a; JSON.stringify(a); } catch (err) { console.log(err.message); }
```